test/parallel/test-quic-send-fd.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const common = require('../common');
+if (!common.hasQuic)
+  common.skip('missing quic');
+
+const assert = require('assert');
+const quic = require('quic');
+const fs = require('fs');
+
+const fixtures = require('../common/fixtures');
+const key = fixtures.readKey('agent1-key.pem', 'binary');
+const cert = fixtures.readKey('agent1-cert.pem', 'binary');
+const ca = fixtures.readKey('ca1-cert.pem', 'binary');
```

Comment:
todo for later... it's so common for us to read all three of these together we can simplify things a bit by adding an additional fixture method... e.g.

something like...
```js
const { key, cert, ca } = fixtures.readTLSKey('agent1-key.pem',
											  'agent1-cert.pem',
											  'ca1-cert.pem');
```